README.md

```diff
@@ -1,19 +1,311 @@
+{ 
+   "taxCheckCode" : "YZFMDJYY9",
+   "ggCredId" : "testCred1",
+   "licenceType": "OperatorOfPrivateHireVehicles",
+   "verifier": {
+      "dateofbirth": "19221201"
+   },
+   "expiresAfter": "20211101",
+   "createDate": "2021-11-12T11:14:26.801+01:00[Europe/London]",
+   "taxCheckStartDateTime": "2021-11-12T11:14:26.801+01:00[Europe/London]",
+   "isExtracted": true,
+   "source": "Digital"
+}
+```
+
+### `GET        /hec/test-only/tax-check/:taxCheckCode`
+Unauthenticated test-only endpoint to retrieve tax check data for the given tax check code. If the tax check code
+exists in the database a `200` (OK) response is returned. If no tax check data can be found for the tax check code, a 
+`401` (NOT FOUND) response is given.
+
+Example response body:
+```json
+{
+   "taxCheckData": {
+      "applicantDetails": {
+         "ggCredId": "credId",
+         "crn": "1234567890",
+         "companyName": "Test Company"
+      },
+      "licenceDetails": {
+         "licenceType": "OperatorOfPrivateHireVehicles",
+         "licenceTimeTrading": "TwoToFourYears",
+         "licenceValidityPeriod": "UpToOneYear"
+      },
+      "taxDetails": {
+         "desCTUTR": "1111111111",
+         "ctStatus": {
+            "ctutr": "1234567890",
+            "startDate": "20201001",
+            "endDate": "20193009"
+         },
+         "recentlyStaredTrading": "Yes"
+      },
+      "taxCheckStartDateTime": "2021-12-02T14:56:11.75Z[Europe/London]",
+      "source": "Digital",
+      "type": "Company"
+   }
+}
+```
+
+### `DELETE     /hec/test-only/tax-check`              
+Unauthenticated test-only endpoint to delete all tax check data. Returns a `200` (OK) response if all data has been
+successfully been deleted.
+
+### `DELETE     /hec/test-only/tax-check/:taxCheckCode`
+Unauthenticated test-only endpoint to delete tax check data corresponding to the given tax check code. Returns a `200` 
+(OK) response if the tax check data has been successfully been deleted.
+
+## HEC Tax Check Data File Transfer
+
+This service will periodically send tax check data to consuming downstream systems via a file transfer process. 
+`HecTaxCheckScheduleService` is the service which takes care of the job scheduling which perform the following tasks:
+1. Identify the extraction time from the config.
+2. TimeCalculator identifies how much time is left for the job to run and the job is scheduled for that time.
+3. At the time of job run, a lock is created on mongo and the following operations are performed 
+   1. Generate all enum lookup files, store them in `object-store` and notify downstream systems that the files are 
+      ready to pick up.
+   2. Find all tax checks which haven't been sent yet (`isExtracted = false`) and generate tax check data file,
+      store in object store and notify downstream systems that the file is ready to pick up.      
+
+One the downstream system has been notified that the files are ready to pick up we expect a callbacks to be made 
```

Comment:
```suggestion
Once the downstream system has been notified that the files are ready to pick up we expect a callback to be made 
```

Reply:
updated [here](https://github.com/hmrc/hec/pull/53/commits/df753d3671d7cf27316ccf0ed61d0c486f887a3b). I kept "callbacks" plural since we expect multiple callbacks per file.

Reply:
👍 in that case, it should be `we expect callbacks` instead of `we expect a callbacks`, but I'm not fussed!

Reply:
yes, you're right :see_no_evil: thanks! I corrected it now [here](https://github.com/hmrc/hec/pull/53/commits/a0d029791e7fd2bb08979851dab803a63802efb0)